server/app/views/applicant/ProgramIndexView.java

```diff
@@ -121,6 +121,25 @@ public Content render(
         request, personalInfo, messages, bundle, /* includeAdminLogin= */ true, applicantId);
   }
 
+  public Content renderWithoutApplicant(
+      Messages messages,
+      Http.Request request,
+      ApplicantService.ApplicationPrograms applicationPrograms,
+      ImmutableList<String> selectedCategoriesFromParams) {
+    return render(
+        messages,
+        request,
+        Optional.empty()
+        /* applicantId= */ ,
```

Comment:
This is a nit, but if it's possible to have the comments appear on the same line as the vars that's preferable (maybe use `//` instead of `/*`)